treeherder/perf/exceptions.py

```diff
@@ -12,3 +12,11 @@ class MissingRecords(Exception):
 
 class CannotBackfill(Exception):
     pass
+
+
+class NoFiledBugs(Exception):
+    pass
+
+
+class BugzillaEndpointProblem(RuntimeError):
```

Comment:
This does not sound that well, it can be renamed to `BugzillaEndpointError` or something else